worker/src/RTC/IceServer.cpp

```diff
@@ -12,13 +12,40 @@ namespace RTC
 
 	static constexpr size_t StunSerializeBufferSize{ 65536 };
 	thread_local static uint8_t StunSerializeBuffer[StunSerializeBufferSize];
+	static constexpr size_t MaxTuples{ 8 };
 
 	/* Instance methods. */
 
 	IceServer::IceServer(Listener* listener, const std::string& usernameFragment, const std::string& password)
 	  : listener(listener), usernameFragment(usernameFragment), password(password)
 	{
 		MS_TRACE();
+
+		// Notify the listener.
+		this->listener->OnIceServerLocalUsernameFragmentAdded(this, usernameFragment);
+	}
+
+	IceServer::~IceServer()
+	{
+		MS_TRACE();
+
+		// Here we must notify to the listener about the removal of all handled
+		// tuples and usernameFragment (including the old one if any).
```

Comment:
Utterly cosmetic: Order is comment is not the same as order in code:

* Comment: notify about the removal of *tuples* and *usernameFragment*
* Code: notifies *usernameFragment* first and *tuples*